client/components/LearnerViewDirective.js

```diff
@@ -584,12 +581,7 @@ tie.directive('learnerView', [function() {
             var syntaxErrorIndex = feedback.getSyntaxErrorIndex();
```

Comment:
A boolean sounds better, yep. But I'm also wondering whether this can be computed on the fly -- i.e. have a hasSyntaxError() method on the feedback object and have that check the types of the feedback paragraphs and compute the appropriate boolean?